{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "3.19.8",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:6.8:rc1:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndIncluding": "6.7.2",
            "versionStartIncluding": "6.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "security@openanolis.org",
      "ID": "CVE-2024-24858"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A race condition was found in the Linux kernel's net/bluetooth in {conn,adv}_{min,max}_interval_set() function. This can result in I2cap connection or broadcast abnormality issue, possibly leading to denial of service."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-362"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://bugzilla.openanolis.cn/show_bug.cgi?id=8154",
          "refsource": "",
          "tags": [
            "Permissions Required"
          ],
          "url": "https://bugzilla.openanolis.cn/show_bug.cgi?id=8154"
        },
        {
          "name": "https://bugzilla.openanolis.cn/show_bug.cgi?id=8154",
          "refsource": "",
          "tags": [
            "Permissions Required"
          ],
          "url": "https://bugzilla.openanolis.cn/show_bug.cgi?id=8154"
        },
        {
          "name": "https://lists.debian.org/debian-lts-announce/2024/06/msg00017.html",
          "refsource": "",
          "tags": [],
          "url": "https://lists.debian.org/debian-lts-announce/2024/06/msg00017.html"
        },
        {
          "name": "https://lists.debian.org/debian-lts-announce/2024/06/msg00017.html",
          "refsource": "",
          "tags": [],
          "url": "https://lists.debian.org/debian-lts-announce/2024/06/msg00017.html"
        },
        {
          "name": "https://lists.debian.org/debian-lts-announce/2024/06/msg00020.html",
          "refsource": "",
          "tags": [],
          "url": "https://lists.debian.org/debian-lts-announce/2024/06/msg00020.html"
        },
        {
          "name": "https://lists.debian.org/debian-lts-announce/2024/06/msg00020.html",
          "refsource": "",
          "tags": [],
          "url": "https://lists.debian.org/debian-lts-announce/2024/06/msg00020.html"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "HIGH",
        "attackVector": "ADJACENT_NETWORK",
        "availabilityImpact": "HIGH",
        "baseScore": 5.3,
        "baseSeverity": "MEDIUM",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:A/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "version": "3.1"
      },
      "exploitabilityScore": 1.6,
      "impactScore": 3.6
    }
  },
  "lastModifiedDate": "2025-02-13T18:17Z",
  "publishedDate": "2024-02-05T08:15Z"
}